Library/Homebrew/cmd/install.rb

```diff
@@ -255,17 +256,17 @@ def install
 
     return if formulae.empty?
 
-    Install.perform_preinstall_checks
+    perform_preinstall_checks
```

Comment:
I like knowing what module this came from.

Reply:
See https://github.com/Homebrew/brew/pull/8084#discussion_r460737938.